include/swift/Parse/ParsedRawSyntaxRecorder.h

```diff
@@ -103,8 +115,8 @@ class ParsedRawSyntaxRecorder final {
   /// For a deferred layout node \p node, retrieve the number of children.
   size_t getDeferredNumChildren(const ParsedRawSyntaxNode &node) const;
 
-#ifndef NDEBUG
-  static void verifyElementRanges(ArrayRef<ParsedRawSyntaxNode> elements);
```

Comment:
I believe removing `static` is a leftover from something you experimented.

Reply:
Oh, yeah. That belonged to the PR which removed `SyntaxKind`.